README.md

```diff
@@ -1,20 +1,102 @@
-# android-boilerplate
+# ultimate-logger-jvm
+[![](https://jitpack.io/v/ArturBorowy/ultimate-logger-jvm.svg)](https://jitpack.io/#ArturBorowy/ultimate-logger-jvm)
 
-Architecture: **MVP**.
+**UltimateLogger** is a library with many useful and customizable logging options.
```

Comment:
I'm just thinking - "library with many useful and customizable sth." is what you can read everywhere and people can loose interest in content in a few seconds in general, so why not save them from reading the whole Readme and state in the first sentence that it let's you log line numbers and method names. If someone is interested, he will surely proceed to details below.
You know, you don't say "I have made a very good smartphone - it has
- ...
- ...
- ...
- ...
- ...
Etc
You write "new phone with 1000gb ram and 100 core CPU", even if it has far more interesting features.

Reply:
I checked, popular Android libs uses same README structure, so I leave it like that. If sombody doesn't want to read about basics, he has big header informing about advenced options, so he will find that he want.